community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -283,7 +282,7 @@ def get_node_action(nodename: str) -> NodeAction:
     elif (state is None or "POWERED_DOWN" in state.flags) and inst.status == "RUNNING":
         log.info("%s is potential orphan node", nodename)
         threshold = timedelta(seconds=90)
-        age = datetime.now() - inst.creation_timestamp
+        age = datetime.now(timezone.utc) - inst.creation_timestamp
```

Comment:
I'm assuming `inst.creation_timestamp` is in Zulu time? Does old datetime auto parse this as UTC?

Reply:
No, it was also parsed with timezone, but prior computation of delta has been done through conversion to timestamp (float epoch seconds).

Reply:
Ah I didn't realize it part of a dataclass with a parsed timestamp, thanks!